assets/scss/components/_tabs.scss

```diff
@@ -70,3 +70,19 @@ $tabBorderWidth: 3px;
 		border-right: $tabBorderWidth solid;
 	}
 }
+
+.tabs-tab--selectedLeft {
```

Comment:
A useful pattern for thinking about classnames is how they can be composed together, e.g. having a 'base class' that gets combined with other classes to 'tune' the style. In the future, a better approach would be to _always_ expect a `tabs-tab--selected` class to be applied to _any_ selected tab, with an _additional_ class of `tabs-tab--selectedLeft` in order to provide 'left' overrides. In that case, you would be able to avoid duplicating the styles in `.tabs-tab--selected`, and just provide the overrides.

cc @eilinora in case you're doing similar updates in the future